lib/s3middleware/nullStream.js

```diff
@@ -0,0 +1,42 @@
+const Readable = require('stream').Readable;
+
+
+/**
+ * This class is used to produce zeros filled buffers for a reader consumption
+ */
+class NullStream extends Readable {
+
+    /**
+     * Construct a new zeros filled buffers producer that will
+     * produce as much bytes as specified by the range parameter, or the size
+     * parameter if range is null or not constituted of 2 elements
+     * @constructor
+     * @param {integer} size - the number of null bytes to produce
+     * @param {array} range - a range specification to override to size
+     */
+    constructor(size, range) {
+        super({});
```

Comment:
Just curious, why pass empty object here?

Reply:
To be consistent with other stream specializations across all the projects